tests/integration_tests/tests/agentless_tests/test_workflow.py

```diff
@@ -57,6 +57,27 @@ def test_execute_operation(self):
         outputs = self.client.deployments.outputs.get(deployment.id).outputs
         self.assertEquals(outputs['ip_address'], '')
 
+    def test_restart(self):
+        dsl_path = get_resource('dsl/basic.yaml')
+        blueprint_id = self.id()
+        deployment, _ = self.deploy_application(
+            dsl_path,
+            blueprint_id=blueprint_id,
+            timeout_seconds=15
+        )
+        execution = self.execute_workflow('restart', deployment.id)
+        events = self.client.events.list(
+            execution_id=execution.id, include_logs=True)
+        self.assertTrue(
```

Comment:
maybe you should add ordering assert, that the stopping happened before the starting